control-plane/pkg/reconciler/trigger/trigger.go

```diff
@@ -488,3 +489,8 @@ func deliveryOrderFromString(val string) (contract.DeliveryOrder, error) {
 		return contract.DeliveryOrder_UNORDERED, fmt.Errorf("invalid annotation %s value: %s. Allowed values [ %q | %q ]", deliveryOrderAnnotation, val, sources.Ordered, sources.Unordered)
 	}
 }
+
+func isGroupAuthorizationError(err error) bool {
```

Comment:
I like to filter "auth" issues out, for a smooth transition towards the actual upstream logic in 1.13